### PR TITLE
Refactor button styles into its own partial - Closes #91

### LINKED ISSUE #91: style Gear Up display section

### PR DESCRIPTION
**Pull request by kccrs**
## Purpose
This PR refactors and consolidates button and dropdown styles into a new partial to ensure CSS for buttons is DRY.

## Approach
I updated the mixin to include width and font size as arguments since many buttons had these defined.  Also created a partial and pulled all buttons and dropdowns out of other partials to consolidate styles.

## Pre-merge TODOS:
None - this is ready to merge pending code review

### CODE REVIEW

File: lib/components/Admin.js [hunk 1]
```diff
@@ -151,14 +152,20 @@ export default class Admin extends Component {
         <section className='SortSpikes'>
           <p className='SortOptions'>Sort Spikes by:
             <label htmlFor='SpikeStatusSort' aria-label='Sort spikes by status'>
-              <select id='SpikeStatusSort' name='all' onChange={(e)=>this.setState({ filter: e.target.value })}>
+              <select i
```
Comment (Jeff-Duke) [inline lib/components/Admin.js]: This is weirdly broken up.. does it not cause issues to have a line break between i and d in 'id' ?

**Comment by Peter-Springer** [comment]
Some inconsistency from "Our Guest Speaker This Week:" and "Gear Up" missing colon. Which one do you guys prefer?

**Comment by Jeff-Duke** [comment]
This branch introduces aXe errors for the add-spike button on the user page, it looks like it's the old, lighter color and doesn't have sufficient contrast.

**Comment by Jeff-Duke** [comment]
I'm not seeing the location or approval status on any of our spikes.

**Comment by Jeff-Duke** [comment]
Merging this branch, opening an issue for media query

**Pull request merged**